tests-cuda/test_1270-install-test.py

```diff
@@ -3,6 +3,7 @@
 import pytest  # noqa: F401
 
 
+@pytest.mark.skip("We don't use shared library for accessing CUDA Kernels anymore.")
```

Comment:
Rather than skipping, this test can be removed. We'd skip it if our intention was to reinstate it in the future, but if doing everything through CuPy is the right way to go (and it looks like it is), then we'd never have this library and wouldn't have to test loading it.